fastlane/metadata/android/ko-KR/full_description.txt

```diff
@@ -1,24 +1,25 @@
-Status는 오픈 소스인 이더리움 기반 앱입니다. 안드로이드 기기로 Dapps의 혁신적인 세계에 접속해 채팅하며 거래할 수 있도록 도와드립니다.
+스테이터스(Status)는 이더리움에 기반한 오픈소스 어플리케이션으로서, 안드로이드 기기로 이더리움 네트워크를 통해 다른 사람들과 채팅하거나, 암호화폐를 주고 받고, 혁신적인 분산앱(DApps)들을 브라우징 할 수 있도록 돕습니다.
 
-<b>중요:</b> Status는 현재 베타 테스팅 중입니다. 베타 버전으로 테스팅함으로 인한 위험이 존재합니다. 따라서 베타 테스팅에 참여하시는 경우 이러한 위험을 이해하고 동의한다는 내용을 읽고 인정하는 것으로 표시됩니다.
 
-여전히 앱을 개발하고 있는 도중이지만 베타 서비스를 통해 이더리움과 친숙해지고 직관적으로 이해할 수 있도록 도와드리고자 하며, 이더리움 네트워크를 위한 최고의 클라이언트로 성장하고자 합니다.
+<b>중요:</b> 스테이터스는 현재 베타 테스팅 중입니다. 베타 버전 테스터로 참여하실 경우, 서비스 이용 중에 프로그램 상의 오류나 위험요소가 있을 수 있습니다. 따라서 베타 테스팅에 참여하시는 경우, 이러한 내용을 이해하고 동의한다는 내용을 숙지했으며 인정하는 것으로 간주됩니다.
 
-<b>Status는 무엇일까요?</b>
-새로운 종류의 인터넷을 위한, 새로운 종류의 도구입니다. 인터넷 서핑, 채팅, 거래, 그 모든 것을 하나의 앱에서 만나보세요. 
+스테이터스는 베타 서비스를 통해 사람들이 이더리움과 조금 더 친숙해지고 직관적으로 이해할 수 있도록 도우며, 이더리움 네트워크 최고의 클라이언트로 도약 할 것입니다.
 
-사람들이 네트워크를 확장하는 데 기여하고자 창조하고 제시한 가치를 스스로 통제할 수 있는 세계를 보고 싶습니다. 이를 위해 새로운 종류의 사람간 상호작용과 경제 활동에 참여하고 이로 인한 혜택을 누리기 쉽도록 도와드릴 도구를 제작했습니다.
+<b>스테이터스는 무엇인가요?</b>
+스테이터스는 차세대 인터넷을 위한 새로운 도구입니다. 브라우징, 채팅, 거래, 모든 것을 하나의 앱에서 체험하실 수 있습니다. 
+
+우리는 사람들이 네트워크를 확장하는 데 적극적으로 참여하며 가치를 창조하고, 창조한 가치를 스스로 관리할 수 있는 세계를 지향합니다. 이것을 실현하기 위해, 스테이터스는 유용한 도구를 개발하여 사람들이 새로운 커뮤니케이션 방식을 통해 디지털 경제 활동에 참여하고, 이로 인한 혜택을 누리는 것을 돕습니다.
 
 <B>베타 서비스 기능</b>
 
-■ 가벼운 클라이언트 — 가벼운 클라이언트와 위스퍼 프로토콜을 사용하여 Go Ethereum의 모든 필수 요소를 갖춘 Status를 사용해 보세요.
+■ 라이트 클라이언트 — 스테이터스는 라이트 클라이언트(Light Client)와 위스퍼(Whisper) 프로토콜을 사용하여 Go Ethereum의 모든 것을 구동합니다.
 
-■ 메시지 암호화 — 위스퍼 프로토콜을 기반으로 구축된 메시지 시스템에서 분산화된 일대일 채팅, 그룹 채팅, 소셜 페이먼트를 사용해 보세요.
+■ 메시지 암호화 — 위스퍼 프로토콜을 기반으로 구축되어 완벽히 암호화되고 분산화된 1:1 채팅, 그룹 채팅, 소셜 결제 기능을 사용해 보세요.
 
-■ Dapps 둘러보기 — Status' Web 3.0 브라우저를 사용하시면 현재 대두되고 있는 Dapps의 다양한 기능을 바로바로 이용하실 수 있습니다.
+■ 분산앱(DApps) 브라우징 — 스테이터스의 Web 3.0 브라우저를 활용하여 다양한 분산앱을 매끄럽게 브라우징 할 수 있습니다. 
 
-■ 이더 월렛 — ‘기본’ 지갑으로 설정된 이더 월렛을 통해 이더리움 저장소. 판매/구매 거래, QR 리더기, 거래 기록을 만나보세요.
+■ 이더 지갑 (Ether Wallet) — 디폴트로 설정된 이더 지갑을 통해 ETH 스토리지, 암호화폐 입출금, QR 리더기, 거래 기록을 체험해보세요.
 
 
 <b>참여하기</b>
-Status를 다운받으시고 테스트하시면 오픈 소스 프로젝트로 개발에 참여하실 수 있습니다. 저희의 사명은 새로운 패러다임인 이더리움을 통해 인터넷과 금융 시스템이 더 개방적으로 변모할 수 있도록 하는 것입니다. Status의 더 높은 목표는 인간으로서 사생활에 관한 집단적인 권리를 보존하고 검열의 위험을 낮추며 경제적인 거래를 더욱 투명하고 오픈된 방식으로 이루어지도록 하는 것입니다. 이를 위해 모두가 참여할 수 있고 변화를 이끌어가는 원인이 될 수 있는 커뮤니티를 구축하고 있습니다. 여러분의 연락을 기다리고 있습니다. Https://chat.status.im/에 가입해 주세요.
\ No newline at end of file
+어플리케이션을 다운로드 받아 테스트하며, 스테이터스 오픈소스 프로젝트에 참여하실 수 있습니다. 우리의 비전은 이더리움이 제시하는 새로운 인터넷 패러다임을 통해 인터넷과 금융 시스템이 조금 더 개방적으로 변모할 수 있도록 하는 것입니다. 스테이터스의 궁극적 목표는 사람들의 소중한 프라이버시를 지키고 검열로부터 보호하며, 더욱 투명하고 개방된 방식의 거래를 촉진하는 것입니다. 이를 위해 스테이터스는 누구든지 참여해 혁신적인 변화를 이끌어가는 커뮤니티를 구축하고 있습니다. 여러분의 연락을 기다리고 있습니다. Https://chat.status.im/에 참여하세요!
```

Comment:
Thanks jinhojang93! 
I'm thinking about whether `Https://chat.status.im/에 참여하세요!` should be with whitespace in URL (so `https://chat.status.im/ 에 참여하세요!`) because Https://chat.status.im/에 resolves as a single link and so leads to 404th response)

Reply:
We apparently had it for a long time and it's a nice chance to update it as well :)

Reply:
Good catch @Serhy!